spring-web/src/main/java/org/springframework/http/HttpHeaders.java

```diff
@@ -502,6 +502,9 @@ public void setAcceptLanguage(List<Locale.LanguageRange> languages) {
 	 */
 	public List<Locale.LanguageRange> getAcceptLanguage() {
 		String value = getFirst(ACCEPT_LANGUAGE);
+		if (value != null) {
```

Comment:
While I think a null check is fine, why not also validate whether this string in non empty and at least [n] chars?